docs/src/content/docs/guides/components.mdx

```diff
@@ -195,3 +195,49 @@ The code above generates the following on the page:
 A list of all available icons is shown below with their associated names. Click an icon to copy the component code for it.
 
 <IconsList />
+
+### Code
+
+Starlight uses Expressive Code to render [Markdown and MDX code blocks](/guides/authoring-content/#code-blocks). Use the `<Code>` component to render syntax highlighted code when using a Markdown code block is not possible, for example, to render data coming from external sources like files, databases, or APIs.
```

Comment:
```suggestion
Use the `<Code>` component to render syntax highlighted code when using a [Markdown code block](/guides/authoring-content/#code-blocks) is not possible, for example, to render data coming from external sources like files, databases, or APIs.
```
I'm having a hard time putting my finger on it, but it seemed weird to state Expressive Code here without linking to it (but then there was immediately an important link), and the fact it was at the very beginning of the sentence/introduction seemed like it was "the main point."

I think this might get to the point, and still retain all the useful information?